Dockerfile

```diff
@@ -6,7 +6,7 @@
 # Stop               : press ctrl + c 
 #                                     or 
 #                                        docker stop mermaid-live-editor
-FROM node:16.12.0 as mermaid-live-editor-builder 
+FROM node:17.0.1 as mermaid-live-editor-builder 
```

Comment:
Should we stick with LTS?

Reply:
Why not but I'm not sure if it is possible to manage it to dependabot 🤔

Reply:
I think we should stick to hard coded versions, because the changes throughout versions may not be supported by packages.